tantalus/migrations/0101_auto_20190122_2303.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.14 on 2019-01-22 23:03
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from tantalus.models import DNALibrary
+
+
+class Migration(migrations.Migration):
+    def update_result_library_id(apps, schema_editor):
+        results = apps.get_model('tantalus', 'ResultsDataset')
+
+        for result in results.objects.all():
+            if result.analysis :
+                if isinstance(result.analysis.args,dict) and 'library_id' in result.analysis.args:
+                    for library in DNALibrary.objects.filter(sequencedataset__id__in=result.analysis.input_datasets.all()):
+                        result.libraries.add(library.id)
+                        result.save
```

Comment:
result.save will not do anything since its a method, but you dont need it anyway.